intelmq/bots/experts/domaintools/expert.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""
+domaintools expert: query domtaintools.com to get a reputation score for a domain name
+
+"""
+from intelmq.lib.bot import Bot
+try:
+    from domaintools import API, exceptions
+except ImportError:
+    API = None
+
+
+class DomaintoolsExpertBot(Bot):
+
+    def init(self):
+        self.logger.info("Loading Domaintools expert.")
+        if (not API):
+            self.logger.exception("need to have the domaintools API installed. See https://github.com/domaintools/python_api")
+            self.stop()
```

Comment:
Wrong usage of `logger.exception`. It prints the traceback of a fromerly catched exception.

Replace both lines with `raise ValueError("your error message")`. The bot class performs the stop itself.